11.0.Dockerfile

```diff
@@ -41,6 +42,7 @@ ONBUILD ENV ADMIN_PASSWORD="$ADMIN_PASSWORD" \
             PGHOST="$PGHOST" \
             PGPORT=$PGPORT \
             PGDATABASE="$PGDATABASE" \
+            DBFILTER="$DBFILTER" \
```

Comment:
The onbuild+env system to set variables is getting deprecated. Better just add a normal `ENV` below, somewhere around lines 65-85.